src/main/java/org/folio/circulation/domain/representations/ItemSummaryRepresentation.java

```diff
@@ -32,6 +32,9 @@ public JsonObject createItemSummary(Item item) {
     write(itemSummary, "barcode", item.getBarcode());
     write(itemSummary, "contributors", item.getContributorNames());
     write(itemSummary, "callNumber", item.getCallNumber());
+    write(itemSummary, "enumeration", item.getEnumeration());
```

Comment:
The `ItemSummaryRepresentation.java` class is used by various endpoints, e.g.: requests, reports, etc., in a fact they all should be updated as well, or an custom representation should be introduced for loans only (I don't think it make big sense, because we're applying similar changes in https://github.com/folio-org/mod-circulation/pull/429/files to hold-shelf report and requests).

Reply:
Done.

Reply:
This topic suggests that other schemas should have changed, and it is marked resolved. Have the changes made to the other schema's been pushed and they don't appear to be in the pull request at the moment?